modules/administration/pages/monitoring.adoc

```diff
@@ -0,0 +1,6 @@
+[[monitoring]]
+= Monitoring
+
+{productname} supports third party monitoring with Icinga or Prometheus.
```

Comment:
Even if PM decides do keep a reference to Icinga to support "legacy" use cases, I would frame the Prometheus support in separate.

In case we want to keep Icinga references, it would be something to the effect of "here is what you can do in case you want to plug-in your existing Icinga" versus a complete monitoring solution provided by us on top of Prometheus.